core/cas-server-core-audit-api/src/main/java/org/apereo/cas/audit/spi/principal/ChainingAuditPrincipalIdProvider.java

```diff
@@ -52,4 +52,9 @@ public String getPrincipalIdFrom(final Authentication authentication, final Obje
     public boolean supports(final Authentication authentication, final Object resultValue, final Exception exception) {
         return true;
     }
+
+    @Override
```

Comment:
Remove, add @Setter?

Reply:
Spotbugs didn't like that. It wants a static field.